site/wesselversluis/data.tsv

```diff
@@ -0,0 +1,8 @@
+day	frequency
```

Comment:
This looks great, but you’re uploading files to `site/wesselversluis/`, but they should go in `site/class-1-bar/wesselversluis/`.

Reply:
Ah oké! Wel in mijn eigen repository of op die van de course? Moet ik deze pull request dan verwijderen en even een nieuwe aanmaken?

Reply:
It’s probably simplest to remove this branch, making a new branch, and trying once more!